nixos/modules/programs/fzf.nix

```diff
@@ -30,7 +30,7 @@ in
     '';
 
     programs.fish.interactiveShellInit = ''
-      eval "$(${getExe pkgs.fzf} --fish)"
+      ${getExe pkgs.fzf} --fish | source
```

Comment:
Should we do this for the others as well?

Reply:
I think it's just the [`fish` setup that is different](https://github.com/junegunn/fzf?tab=readme-ov-file#setting-up-shell-integration)
<img width="437" alt="image" src="https://github.com/NixOS/nixpkgs/assets/65531/19cfd446-9667-45eb-a80a-a2e9bca39e9b">